src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,49 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Parser {
+    public static String[] inputParser(String input) {
+        String[] processed = input.split(" ", 2);
+        if (processed.length != 2) {
+            processed = new String[]{processed[0], ""};
+        }
+        return processed;
+    }
+
+
+    public static Command commandParser(String input) {
+        for (Command c : Command.values()) {
+            if (c.value.equals(input)) {
+                return c;
+            }
+        }
+        return Command.UNKNOWN;
+    }
+
+    public static String dateParser(String date) {
```

Comment:
```suggestion
    public static String parseDate(String date) {
```
Similarly, you may want to name methods in verb form. I noticed some other occurrences of this too.